pkg/ctl/cmdutils/nodegroup.go

```diff
@@ -18,26 +18,21 @@ const (
 
 // AddCommonCreateNodeGroupFlags adds common flags for creating a node group
 func AddCommonCreateNodeGroupFlags(cmd *cobra.Command, fs *pflag.FlagSet, p *api.ProviderConfig, cfg *api.ClusterConfig, ng *api.NodeGroup) {
-	var desiredCapacity int
-	var minSize int
-	var maxSize int
-
 	fs.StringVarP(&ng.InstanceType, "node-type", "t", defaultNodeType, "node instance type")
-	fs.IntVarP(&desiredCapacity, "nodes", "N", api.DefaultNodeCount, "total number of nodes (for a static ASG)")
 
-	// TODO: https://github.com/weaveworks/eksctl/issues/28
-	fs.IntVarP(&minSize, "nodes-min", "m", api.DefaultNodeCount, "minimum nodes in ASG")
-	fs.IntVarP(&maxSize, "nodes-max", "M", api.DefaultNodeCount, "maximum nodes in ASG")
+	desiredCapacity := fs.IntP("nodes", "N", api.DefaultNodeCount, "total number of nodes (for a static ASG)")
```

Comment:
@mumoshu just FYI. I figured this was a little nicer without having to have the `var` block.

Reply:
Neat! Thanks for sharing, and your work.